multiversx_sdk_cli/contracts.py

```diff
@@ -211,26 +210,16 @@ def query_contract(self,
         query_runner = QueryRunnerAdapter(proxy)
         sc_query_controller = SmartContractQueriesController(query_runner, self._abi)
 
-        query = sc_query_controller.create_query(
-            contract=contract_address.to_bech32(),
-            function=function,
-            arguments=args
-        )
-
-        response = sc_query_controller.run_query(query)
-
-        if self._abi:
-            return sc_query_controller.parse_query_response(response)
-        else:
-            return [self._query_response_to_dict(response)]
+        try:
+            response = sc_query_controller.query(
```

Comment:
So we don't support the `caller` and `callValue` parameters? But we didn't support them before anyway. I think we should, in the future.

Reply:
we can do that in the future, we didn't support them before.